code-studio/build-js.js

```diff
@@ -31,6 +36,16 @@ build_commands.execute([
     commonFile: 'code-studio-common',
     shouldMinify: commander.min,
     shouldWatch: commander.watch
+  }),
+  build_commands.browserify({
+    srcPath: SRC_PATH + 'initApp/',
+    buildPath: BUILD_PATH,
+    filenames: [
+      'initApp.js'
```

Comment:
After all of our talking about naming the output as shared.js, I realized that name was not nearly specific enough once this is inside of code-studio. Open to suggestions for better names than initApp, as really this covers more than that (i.e. it has a bunch of project stuff).